nvme-ioctl.c

```diff
@@ -867,3 +867,15 @@ int nvme_self_test_start(int fd, __u32 nsid, __u32 cdw10)
 
 	return nvme_submit_admin_passthru(fd, &cmd);
 }
+
+int nvme_virtual_mgmt(int fd, __u32 cdw10, __u32 cdw11)
```

Comment:
This one should take a __u32 *result field since the virtualization mgmt command does return something informational in CQE DW0, and if successful, we want to print that out. See nvme_get_feature, as an example of what I mean.